test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/FhirStorageTests.cs

```diff
@@ -1019,14 +1028,24 @@ private async Task SetAllowCreateForOperation(bool allowCreate, Func<Task> opera
             }
         }
 
-        private ResourceWrapper CreateObservationResourceWrapper(string observationId)
+        private ResourceWrapper CreateObservationResourceWrapper(string observationId, bool setRawResourceNull = false)
         {
             Observation observationResource = Samples.GetDefaultObservation().ToPoco<Observation>();
             observationResource.Id = observationId;
             observationResource.VersionId = "1";
 
             var resourceElement = observationResource.ToResourceElement();
-            var rawResource = new RawResource(observationResource.ToJson(), FhirResourceFormat.Json, isMetaSet: true);
+            RawResource rawResource;
+
+            if (setRawResourceNull)
+            {
+                rawResource = new RawResource(FhirResourceFormat.Json, isMetaSet: true);
```

Comment:
I'm not thrilled with having a constructor just for a single unit test. Could we use a Substitute here instead?